regression/scripts/Vehicles/Cessna_172.py

```diff
@@ -0,0 +1,222 @@
+# Cessna_172.py
+#
+# Created:  Feb 2017, M. Vegh (modified from data originally in cmalpha/cmalpha.py)
+# Modified: 
+
+""" setup file for the Beech 99 aircraft, current values only used to test stability cmalpha
+"""
+
+# ----------------------------------------------------------------------
+#   Imports
+# ----------------------------------------------------------------------
+
+import numpy as np
+import SUAVE
+from SUAVE.Core import Units
+from SUAVE.Core import (
+    Data, Container,
+)
+from SUAVE.Methods.Geometry.Three_Dimensional.compute_span_location_from_chord_length import compute_span_location_from_chord_length
+from SUAVE.Methods.Flight_Dynamics.Static_Stability.Approximations.datcom import datcom
+from SUAVE.Methods.Flight_Dynamics.Static_Stability.Approximations.Supporting_Functions.trapezoid_ac_x import trapezoid_ac_x
+
+def vehicle_setup():
+    vehicle = SUAVE.Vehicle()
+    GTOW    = 2200. * Units.pounds     
+    vehicle.mass_properties.max_takeoff  = GTOW #from Wikipedia
+    vehicle.mass_properties.empty        = 2515*Units.kg
+    vehicle.mass_properties.max_zero_fuel= vehicle.mass_properties.max_takeoff-vehicle.mass_properties.empty+15.*225*Units.lbs #15 passenger ac
+    vehicle.envelope.ultimate_load       = 3.8    
+    vehicle.envelope.limit_load          = 1.           
+    #define fuel weight needed to size fuel system
+    fuel                                                = SUAVE.Attributes.Propellants.Aviation_Gasoline()
+    fuel.mass_properties                                = SUAVE.Components.Mass_Properties()
+    fuel.mass_properties.mass                           = 252 *Units.lbs
+    fuel.number_of_tanks                                = 1.
+    fuel.internal_volume                                = fuel.mass_properties.mass/fuel.density #all of the fuel volume is internal
+    vehicle.fuel                                        = fuel
+    
+    
+    propulsors = SUAVE.Components.Propulsors.Propulsor() #use weights for the IC engine  
+    propulsors.tag = 'internal_combustion'
+    propulsors.rated_power = 110 *Units.kW # engine correlation is really off
+    #propulsors.rated_power = 100 *Units.kW
```

Comment:
Delete this line. Also what's the issue with the engine correlation?

Reply:
It doesn't really match the weight breakdown from Roskam, which is where I got some detailed weight statements. There are a number of possible reasons why, one of which might be different ways of bookkeeping things

Reply:
Okay that's fine. If you could just write something like that in the comment I think it would be great.